src/grammar.json

```diff
@@ -6,19 +6,19 @@
       "type": "REPEAT",
       "content": {
         "type": "SYMBOL",
-        "name": "_definition"
```

Comment:
I'm assuming that other than random spot checks, we can ignore diffs under `src/`, as these are code generated from `grammar.json`.